tests/conftest.py

```diff
@@ -1,12 +1,14 @@
 
-from bbtest import LocalHost, WindowsHost, LinuxHost
+from bbtest import LocalHost, WindowsHost, LinuxHost, OSXHost
 from tests.test_os_agnostic import BaseToDoTest
 
 os_2_class = {'local': LocalHost,
               'win': WindowsHost,
               'windows': WindowsHost,
               'win32': WindowsHost,
-              'linux': LinuxHost}
+              'linux': LinuxHost,
+              'mac': OSXHost,
+              'osx': OSXHost}
```

Comment:
"There should be one-- and preferably only one --obvious way to do it."

Please join 'mac' & 'osx' into one entry and call it `posix`  https://stackoverflow.com/a/1879/66595

Reply:
need to further discuss it. maybe requires a more extensive fix.